spec/integration/agent/spec.yml

```
@@ -12,3 +12,7 @@ spec:
       sizeLimit: 2MB
       pattern: '^.+.yml.hbs$'
```

It looks like it was also missing to escape the dots from the extensions in this pattern.

```suggestion
      pattern: '^.+\.yml\.hbs$'
```

---

Created PR to update those regexes that I found in the spec
https://github.com/elastic/package-spec/pull/889